app/assets/stylesheets/user.scss

```diff
@@ -26,31 +26,23 @@
   padding: 0 20px 0 20px;
 }
 
-.breed-box img {
+.image-container {
+  display: flex;
+  align-items: center;
```

Comment:
flex and now centers every image rather than hiking it to the top of the container